tests/cmake/gen_rst.cmake

```diff
@@ -0,0 +1,73 @@
+find_package(CMinx 1.0.0 PATHS /home/branden/git/CMakeDoc/build NO_DEFAULT_PATH REQUIRED CONFIG)
```

Comment:
Should this be `${CMAKE_BINARY_DIR}`?

Reply:
Whoops, guess I accidentally left that in. I thought it was running the test but if that didn't fail it might not be, I'll take a look later today

Reply:
@ryanmrichard appears there's bigger issues, according to the GitHub Action logs all tests actually failed, and it just didn't mark it for some reason. Additionally I don't think the CMake tests are being ran at all. I'll mark this pull request as in progress for the moment while I figure out what's going on

Reply:
This has been fixed now and the tests are being ran by CI